layers/landcover/landcover.sql

```diff
@@ -136,5 +136,6 @@ RETURNS TABLE(osm_id bigint, geometry geometry, class text, subclass text) AS $$
         -- etldoc:  landcover_z14 -> layer_landcover:z14_
         SELECT *
         FROM landcover_z14 WHERE zoom_level >= 14 AND geometry && bbox
-    ) AS zoom_levels;
+    ) AS zoom_levels
+    ORDER BY ST_Area(geometry) DESC;
```

Comment:
Maybe the `area` column could be used directly here, if it's already built by imposm.

Reply:
Sadly `area` is not available during tile generation.

This column is used during computation of generalized layers [here](https://github.com/QwantResearch/openmaptiles/blob/master/layers/landcover/mapping.yaml#L5-L44), but will not be added in the final computation [here](https://github.com/QwantResearch/openmaptiles/blob/master/layers/landcover/landcover.sql#L28-L86). However this change might justify a modification of this behavior if we consider it might affect performances?

Reply:
Just updated how this is computed: https://github.com/QwantResearch/openmaptiles/pull/33/commits/f290fa1c748210032bfb1b2d690e98765f7ffeb9

For most zoom levels we can reuse computed area, otherwise this is done for the ordering.